src/test/java/teammates/ui/webapi/UpdateFeedbackSessionActionTest.java

```diff
@@ -94,6 +99,240 @@ protected void testExecute() {
 
         assertNotNull(response.getCreatedAtTimestamp());
         assertNull(response.getDeletedAtTimestamp());
+
+        Map<String, Long> expectedStudentDeadlines = new HashMap<>();
+        expectedStudentDeadlines.put("student3InCourse1@gmail.tmt", 1809126000000L);
+        expectedStudentDeadlines.put("student4InCourse1@gmail.tmt", 1809126000000L);
+        expectedStudentDeadlines.put("student5InCourse1@gmail.tmt", 1809126000000L);
+        assertEquals(expectedStudentDeadlines, response.getStudentDeadlines());
+        Map<String, Long> expectedInstructorDeadlines = new HashMap<>();
+        expectedInstructorDeadlines.put("instructor1@course1.tmt", 1809126000000L);
+        expectedInstructorDeadlines.put("instructor2@course1.tmt", 1809126000000L);
+        assertEquals(expectedInstructorDeadlines, response.getInstructorDeadlines());
+    }
+
+    @Test
+    public void testExecute_changeDeadlineForStudents_shouldChangeDeadlinesCorrectlyWhenAppropriate() {
+        InstructorAttributes instructor1ofCourse1 = typicalBundle.instructors.get("instructor1OfCourse1");
+        FeedbackSessionAttributes session = typicalBundle.feedbackSessions.get("session1InCourse1");
+
```

Comment:
Do we have any case where we delete/update/create at the same time? I personally think there is value to adding one (debatable).

Reply:
For now, the behavior at the frontend is that you can update/create at the same time, but deletion is separate